samples/features/sql-big-data-cluster/deployment/private-bdc/scripts/deploy-private-bdc.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Get password as input. It is used as default for controller, SQL Server Master instance (sa account) and Knox.
+#
+while true; do
+    read -s -p "Create Admin username for Big Data Cluster: " bdcadmin
+    echo
+    read -s -p "Create Password for Big Data Cluster: " password
+    echo
+    read -s -p "Confirm your Password: " password2
+    echo
+    [ "$password" = "$password2" ] && break
+    echo "Password mismatch. Please try again."
+done
+
+
+# Create BDC custom profile
+azdata bdc config init --source aks-dev-test --target private-bdc-aks --force
+
+# Configurations for private BDC deployment
+azdata bdc config replace -c private-bdc-aks/control.json -j "$.spec.docker.imageTag=2019-CU6-ubuntu-16.04"
+azdata bdc config replace -c private-bdc-aks/control.json -j "$.spec.storage.data.className=default"
+azdata bdc config replace -c private-bdc-aks/control.json -j "$.spec.storage.logs.className=default"
+
+azdata bdc config replace -c private-bdc-aks/control.json -j "$.spec.endpoints[0].serviceType=NodePort"
+azdata bdc config replace -c private-bdc-aks/control.json -j "$.spec.endpoints[1].serviceType=NodePort"
+
+azdata bdc config replace -c private-bdc-aks /bdc.json -j "$.spec.resources.master.spec.endpoints[0].serviceType=NodePort"
+azdata bdc config replace -c private-bdc-aks /bdc.json -j "$.spec.resources.gateway.spec.endpoints[0].serviceType=NodePort"
+azdata bdc config replace -c private-bdc-aks /bdc.json -j "$.spec.resources.appproxy.spec.endpoints[0].serviceType=NodePort"
+
+# In case you're deploying BDC in HA mode ( aks-dev-test-ha profile ) please also use the following command 
+# azdata bdc config replace -c private-bdc-aks /bdc.json -j "$.spec.resources.master.spec.endpoints[1].serviceType= NodePort"
+
+
+export AZDATA_USERNAME=$bdcadmin
+export AZDATA_PASSWORD=$password
+export ACCEPT_EULA=yes  #accept agreement
+
+azdata bdc create --config-profile private-bdc-aks --accept-eula yes
```

Comment:
I suggest to add a step to login and show endpoints once deployment completes, as this would be the next step the customer will attempt.